boefjes/boefjes/plugins/kat_rpki/main.py

```diff
@@ -1,21 +1,31 @@
 """Boefje script for validating vrps records based on code from @trideeindhoven"""
 import json
+import os
 from datetime import datetime
-from typing import List, Tuple, Union
+from os import getenv
+from typing import Bool, Dict, List, Tuple, Union
 
+import requests
 from netaddr import IPAddress, IPNetwork
 
 from boefjes.job_models import BoefjeMeta
 
-FINDING_TYPE_PATH = "boefjes/plugins/kat_rpki/rpki.json"
+RPKI_PATH = "boefjes/plugins/kat_rpki/rpki.json"
+RPKI_META_PATH = "boefjes/plugins/kat_rpki/rpki-meta.json"
```

Comment:
These paths are not guaranteed to be writable. In containers we still run everything as root because of the docker socket, but in the Debian packages we run boefjes as user `kat` and we can't write to those paths.

I think the best place for this would be `/var/cache/kat`, but we would have to create that in both the containers image and Debian packaging. Using such a separate cache directory would also make it possible to mount a shared cache directory there.